error.js

```diff
@@ -1,3 +1,24 @@
-exports.errorMessage = (req, res) => {
+exports.handleInvalidPath = (req, res) => {
   res.status(404).send({ msg: "Sorry, this endpoint does not exist" });
 };
+
+exports.handle404Errors = (err, req, res, next) => {
```

Comment:
Might be worth naming this to something more generic, like handlePSQLErrors, as they won't always be 404